### DISCUSSION TITLE
Full-Page component redirection troubles!

### DISCUSSION THREAD

**Discussion by suryavardhan** [livewire/livewire#6167] [1. Help] [answered]
I made a full page component with a form , which saves a set of information and upon successful submission there is a redirect to another full page component with a flash message. 

As of now , I am able to redirect for the first time with flash message. Then when the user hit 'Add new' button to navigate back to the form component, though the component loads properly, form submission and redirection is not working

Its throwing this error in console

```
livewire.js?id=777821c0:8099 Uncaught (in promise) TypeError: Alpine.navigate is not a function
    at shouldRedirectUsingNavigateOr (livewire.js?id=777821c0:8099:14)
```

**Comment by donkooijman** [comment, marked as answer]
@suryavardhan I had this same issue because I was referencing AlpineJS in a CDN. Maybe it was an old version, I'm not sure. Anyways make sure to check if you have the same. AlpineJS is now included in Livewire, no need to include it in App.js or as a CDN.

![image](https://github.com/livewire/livewire/assets/19776108/f5a816b3-3f83-49e6-bb3d-969b1a997c14)
![image](https://github.com/livewire/livewire/assets/19776108/d40c6e03-f08a-43d9-93fd-fda35598710e)

**Reply by suryavardhan** [reply]
Perfect!  Thanks.
